.travis.yml

```diff
@@ -8,10 +8,22 @@ install:
   - go get -u github.com/golang/lint/golint
   - go get -u github.com/Masterminds/glide
   - go get -u golang.org/x/net/context
+  - go get -u github.com/HewlettPackard/gas
   - export GO15VENDOREXPERIMENT=1
   - glide install
 
 script:
+  # GAS command for SDL scanning
+  - (cd arm && gas -fmt=json -out=results.json ./... > /dev/null)
+  - cat arm/results.json
+  - test $(grep -c "\"severity\":\s*\"HIGH\"" arm/results.json) -lt 8
+  - (cd management && gas -fmt=json -out=results.json ./... > /dev/null)
+  - cat management/results.json
+  - test $(grep -c "\"severity\":\s*\"HIGH\"" management/results.json) -lt 8
```

Comment:
When I ran the scan there was only one finding in the management folder and none in the other folders. So you could make this line (management) "-lt 2" and the others "-lt 1"
To answer Ahmet's question, this is comparing the total number of HIGH findings to the current value to make sure it doesn't increase. We have more elegant tools for parsing incident severity/count internally but those aren't public at this time.